maximum_subarray/Cpp/max_subarray.cpp

```diff
@@ -0,0 +1,28 @@
+#include<bits/stdc++.h>
+
+using namespace std;
+
+int main(){
+	// Reading size of array and array.
+	cin >> n;
+	
+	for(int i=1;i<=n;i++)
+		cin >> arr[i];
+
+
+	for(int i=1;i<=n;i++){
+
+		// Add i'th value to current summary.
+		curSum += arr[i];
+
+		// When current summary becomes negative we should assign zero because adding to negative value isn't logical.
```

Comment:
Lines should be <= 80 characters long  [whitespace/line_length] [2]

*Origin: CPPLintBear, Section: `all.cpplint`.*